task8/TutterApp/src/main/java/models/Constants.java

```diff
@@ -0,0 +1,6 @@
+package models;
+
+public class Constants {
```

Comment:
Need a private constructor here to prevent creating *instance* of this class.
This is by standard